packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -145,64 +150,66 @@ describe('internal-plugin-metrics', () => {
       let clock;
       let saveLatencySpy;
 
-      beforeEach(() => {       
-        clock = sinon.useFakeTimers(); 
-        
+      beforeEach(() => {
+        clock = sinon.useFakeTimers();
+
         saveLatencySpy = sinon.stub(cdl, 'saveLatency');
       });
 
       afterEach(() => {
         clock.restore();
         sinon.restore();
       });
-      
-      it('checks measureLatency with overwrite false', async () => {
+
+      it('checks measureLatency with accumulate false', async () => {
         const key = 'internal.client.pageJMT';
-        const overwrite = false;
+        const accumulate = false;
+
         const callbackStub = sinon.stub().callsFake(() => {
           clock.tick(50);
           return Promise.resolve('test');
         });
 
-        const promise = cdl.measureLatency(callbackStub, 'internal.client.pageJMT', overwrite);
+        // accumulate should be false by default
+        const promise = cdl.measureLatency(callbackStub, 'internal.client.pageJMT');
 
         const resolvedValue = await promise;
         assert.deepEqual(resolvedValue, 'test');
         assert.calledOnceWithExactly(callbackStub);
-        assert.calledOnceWithExactly(saveLatencySpy, key, 50, overwrite)       
+        assert.calledOnceWithExactly(saveLatencySpy, key, 50, accumulate);
       });
 
-      it('checks measureLatency with overwrite true', async () => {
+      it('checks measureLatency with accumulate true', async () => {
         const key = 'internal.download.time';
-        const overwrite = true;
+        const accumulate = true;
         const callbackStub = sinon.stub().callsFake(() => {
           clock.tick(20);
           return Promise.resolve('test123');
         });
 
-        const promise = cdl.measureLatency(callbackStub, 'internal.download.time', overwrite);
+        const promise = cdl.measureLatency(callbackStub, 'internal.download.time', accumulate);
 
         const resolvedValue = await promise;
         assert.deepEqual(resolvedValue, 'test123');
         assert.calledOnceWithExactly(callbackStub);
-        assert.calledOnceWithExactly(saveLatencySpy, key, 20, overwrite)       
+        assert.calledOnceWithExactly(saveLatencySpy, key, 20, accumulate);
       });
 
       it('checks measureLatency when callBack rejects', async () => {
         const key = 'internal.client.pageJMT';
-        const overwrite = true;
+        const accumulate = true;
```

Comment:
in order to keep the test the same as before, we should say accumulate = false, as before it was overwrite = true